naga/src/back/msl/writer.rs

```diff
@@ -721,6 +728,11 @@ impl<'a> ExpressionContext<'a> {
         index::bounds_check_iter(chain, self.module, self.function, self.info)
     }
 
+    /// See docs for `proc::index::oob_locals`.
```

Comment:
nitpick: Let's link where we can; `rustdoc` can take some of the load off in keeping this path correct.

Reply:
Pushed as a `fixup!` with [5208f4e](https://github.com/gfx-rs/wgpu/pull/7323/commits/5208f4ef0da934aebab43c08e39d357db3eefcac).